vertx-web/src/main/java/io/vertx/ext/web/handler/impl/StaticHandlerImpl.java

```diff
@@ -566,6 +575,18 @@ public StaticHandler setHttp2PushMapping(List<Http2PushMapping> http2PushMap) {
     return this;
   }
 
+  @Override
+  public StaticHandler setCompressedContentTypes(Set<String> contentTypes) {
+    if(contentTypes != null) this.compressedContentTypes = new HashSet<>(contentTypes);
```

Comment:
Please make sure your editor is configured with the [Vert.x formatting conventions](https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines) and reformat.